doc/motion_planning_pipeline/launch/motion_planning_pipeline_tutorial.launch

```diff
@@ -6,28 +6,9 @@
   <arg name="planning_plugin" value="ompl_interface/OMPLPlanner" />
   <arg name="planning_adapters" value="default_planner_request_adapters/AddTimeParameterization default_planner_request_adapters/FixWorkspaceBounds default_planner_request_adapters/FixStartStateBounds default_planner_request_adapters/FixStartStateCollision default_planner_request_adapters/FixStartStatePathConstraints" />
 
-  <include file="$(find panda_moveit_config)/launch/planning_context.launch">
```

Comment:
you are removing this because it duplicates the panda move_group.launch settings, correct?

Reply:
Yes. I am unifying the tutorials to all use the demo.launch script